jib-cli/README.md

```diff
@@ -120,7 +121,43 @@ This command follows the following pattern:
 ```
 jib jar --target <image name> path/to/myapp.jar [options]
 ```
+### Quickstart
+1. Create a simple hello world java program in `HelloWorld.java`
```

Comment:
Hmm... the process seems unrealistic and too involved. Maybe we can just take an existing JAR. How about using the Spring Petclinic?

1. Have your JAR (thin or fat) ready. For demonstration, we'll use [Spring Petclinic](https://projects.spring.io/spring-petclinic/)

```
$ git clone https://github.com/spring-projects/spring-petclinic.git
$ cd spring-petclinic
$ ./mvnw package
```

2.
```
$ jib jar ... target/boot-jar.jar
```
(we should be able to use either the boot jar or the original jar, but perhaps we can just use a fat jar.)

3. Run the image and open your browser at http://localhost:8080
```
$ docker run -p 8080:8080 ...
```

If Petclinic seems too heavy, we can take some other hello world sample.

Reply:
You are right. Thanks for the suggestion! It's definitely easier to follow now.